packages/components/src/components/form/Select/index.tsx

```diff
@@ -153,7 +153,6 @@ const selectStyle = (
         ...base,
         padding: 0,
         '& + &': {
-            borderTop: `1px solid ${theme.BG_WHITE_ALT_HOVER}`,
```

Comment:
Was this removed from the design on purpose? I think it is clearer with the separator.